multiuser/keycloak/che-multiuser-keycloak-user-remover/src/main/java/org/eclipse/che/multiuser/keycloak/server/deploy/KeycloakUserRemoverModule.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.multiuser.keycloak.server.deploy;
+
+import com.google.inject.AbstractModule;
+import org.eclipse.che.inject.DynaModule;
+import org.eclipse.che.multiuser.keycloak.server.KeycloakUserRemover;
+import org.eclipse.che.multiuser.keycloak.server.KeycloakUserRemover.RemoveUserListener;
+
+@DynaModule
```

Comment:
As far as I remember we tried to get rid of DynaModules to speed up container start up but I may be wrong.
So, I would see that module installed somewhere here https://github.com/eclipse/che/blob/9f4b49bdaa0eb58f8d9ee006e19b957d24b7c9d2/assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterModule.java#L310

Reply:
@skabashnyuk WDYT?

Reply:
@sleshchenko Maybe need to deprecate `@DynaModule` if you decide to not use it anymore

Reply:
@sleshchenko probably right. Let's replace @DynaModule with explicit entry in `WsMasterModule.java`

Reply:
ok